app/controllers/api/databases.php

```diff
@@ -1167,7 +1175,7 @@ function updateAttribute(
             'required' => $required,
             'default' => $default,
             'array' => $array,
-            'format' => APP_DATABASE_ATTRIBUTE_EMAIL,
+            'format' => APP_DATABASE_ATTRIBUTE_EMAIL
```

Comment:
Let's keep all trailing commas, it makes it easier to add a new item or re-order the existing ones